pkg/snapstore/abs_snapstore.go

```diff
@@ -51,14 +52,15 @@ type ABSSnapStore struct {
 }
 
 type absCredentials struct {
-	BucketName     string `json:"bucketName"`
-	SecretKey      string `json:"storageKey"`
-	StorageAccount string `json:"storageAccount"`
+	BucketName     string  `json:"bucketName"`
+	SecretKey      string  `json:"storageKey"`
+	StorageAccount string  `json:"storageAccount"`
+	URI            *string `json:"uri,omitempty"`
 }
 
 // NewABSSnapStore create new ABSSnapStore from shared configuration with specified bucket
 func NewABSSnapStore(config *brtypes.SnapstoreConfig) (*ABSSnapStore, error) {
-	storageAccount, storageKey, err := getCredentials(getEnvPrefixString(config.IsSource))
+	storageAccount, storageKey, overrideURI, err := getCredentials(getEnvPrefixString(config.IsSource))
```

Comment:
why can't `getCredentials` just return the struct instead constituents of the struct?

Reply:
Noted. Making the change.